File: tests/python/cases/case_137_dead_branch_global_04.py
"""
Task case 137: dead_branch_global_04 false-positive/false-negative stress pattern.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_137_dead_branch_global_04(input_data):
    task_name = 'dead_branch_global_04'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'stress',
        'stage': 'evaluation',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    if False:
        RETAINED_CACHE['case_137'] = payload
    # SAFE: sink exists only in dead branch and never executes.
    return payload['checksum']

File: tests/go/cases/case_137_dead_branch_global_04.go
package escape_tests

var retainedCase137 = []map[string]string{}

func Case137DeadBranchGlobal04(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "dead_branch_global_04",
		"entity": "stress",
		"stage": "evaluation",
		"input": raw,
		"checksum": "dead_branch_global_04:" + raw,
	}
	if false {
		retainedCase137 = append(retainedCase137, payload)
	}
	// SAFE: dead branch retention.
	return payload["checksum"]
}
